jobs/tests/test_api.py

```diff
@@ -21,6 +21,7 @@ async def test_jobs_basic_operations(db):
     assert await count(db, "jobs.job_queue") == 1
     tasks = await jobs.consume(db, 1)
     assert len(tasks) == 1
+    assert tasks[0]["run_at"] is not None
```

Comment:
I think this should work but for some reason it fails. Any idea @jordic

Reply:
It could be, because cte's are run multithreaded.